src/node/members.h

```diff
@@ -54,28 +53,36 @@ namespace ccf
 
     MSGPACK_DEFINE(cert, keyshare);
   };
-
   DECLARE_JSON_TYPE(MemberPubInfo)
   DECLARE_JSON_REQUIRED_FIELDS(MemberPubInfo, cert, keyshare)
 
-  struct MemberInfo : MemberPubInfo
+  struct MemberInfo
   {
+    std::vector<uint8_t> cert; //< DER to match key in Certs table
```

Comment:
Let's talk about this one. Should this be a DER or a PEM? It's currently a DER, and I believe this is (as the comment says) so it matches the key used in the member Certs table. But as far as I can tell we never actually use this, there's no code dependency on these being identical. Instead, this object really only exists to store and present information externally, so perhaps it should store PEMs? As in, if a member makes `query` or `read` that is trying to list the active members, should they see the DER (which seems to be an internal implementation detail), or the PEM with which the member was added?

There was an odd confusion here previously: since `cert` is untyped, it actually held a DER when it was a `MemberInfo` but a PEM whenever we built a `MemberPublicInfo` (from JSON). I'm tempted to make an explicit DER type to avoid this, but a) we get almost the same thing by making all PEMs be explicitly `tls::Pem` and b) we have very few true DERs.